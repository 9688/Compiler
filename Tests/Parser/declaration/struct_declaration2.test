struct{
	int f(){
		return 5;	
	}
	int x;
} Z;