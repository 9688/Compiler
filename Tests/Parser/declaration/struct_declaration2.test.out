<--------TABLES-------->
<---TABLE--->
VARS::

Z   STRUCT $UNNAMED${
vars::

f   FUNC () { BLOCK     RETURN          5 } RETURNED int   


x   int   


types::
}   


TYPES::
char char

double double

int int

void void

<---TABLE--->
VARS::

TYPES::
